src/app/game-state/character.ts

```diff
@@ -374,8 +374,8 @@ export class Character {
       // increase by log2 of whatever is over the softcap
```

Comment:
Can you update this comment? I'm not good enough at math to get what this is doing right away.

Reply:
Oop, yeah that's part of the aptitude fix, forgot the comment there. Also added a local variable for jittering the balance as needed.